lib/simulation/openfisca/mapping/index.js

```diff
@@ -119,18 +119,23 @@ var applyRessources = function(mesAidesEntity, openFiscaEntity, mappingSchema, s
     });
 };
 
-var setNonInjectedPrestationsToZero = function(openFiscaEntity, dateDeValeur) {
+var setNonInjectedPrestationsToZero = function(famille, individus, dateDeValeur) {
     var periods = getPeriods(dateDeValeur);
     var targetPeriods = periods.last12Months.concat([periods.rollingYear]);
-
+    var prestations = _.pick(PRESTATIONS, function(definition) {
```

Comment:
Indeed, good performance improvement.

But the varname is ununderstandable: the prestations are the prestations that are of type `Number`?!